can you fix this?



Run agent-readiness-scorecard check --path custom_components/meraki_ha --format markdown > readiness_report.md
  agent-readiness-scorecard check --path custom_components/meraki_ha --format markdown > readiness_report.md
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.11.14/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.14/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.14/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.14/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.14/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.14/x64/lib
/home/runner/work/_temp/492b772c-5cab-417f-884f-edffdc99044f.sh: line 1: agent-readiness-scorecard: command not found
Error: Process completed with exit code 127.

--- a/.github/workflows/agent-readiness.yaml
+++ b/.github/workflows/agent-readiness.yaml
@@ -1,4 +1,4 @@
-name: Agent Readiness Scorecard
+name: Agent readiness scorecard
 
 on:
   pull_request:
@@ -25,16 +25,18 @@ jobs:
         run: pip install git+https://github.com/brewmarsh/agent-readiness-scorecard.git
 
       - name: Run scorecard
-        run: agent-readiness-scorecard check --path custom_components/meraki_ha --format markdown > readiness_report.md
+        continue-on-error: true
+        run: agent-score score custom_components/meraki_ha --report readiness_report.md
 
       - name: Upload readiness report
+        if: always()
         uses: actions/upload-artifact@v4
         with:
           name: readiness-report
           path: readiness_report.md
 
       - name: Post comment to PR
-        if: github.event_name == 'pull_request'
+        if: always() && github.event_name == 'pull_request'
         uses: marocchino/sticky-pull-request-comment@v2
         with:
           path: readiness_report.md

--- a/.github/workflows/agent-readiness.yaml
+++ b/.github/workflows/agent-readiness.yaml
@@ -12,6 +12,9 @@ on:
 jobs:
   readiness-score:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4

